README.md

```diff
@@ -1,12 +1,94 @@
+**EduHelp** simply mean to make it easier for you to *record any online session*, be it an online lecture, meetings, youtube tutorials, etc. But we are not simply making a web extension or desktop app that records your screen or gives you an audio file in the end. We mean to *save the slide show* or screen presentation going on during those meets as well as the *text file of the entire conversation* that took place. This Project will not only help normal people but also *disabled persons* because the audio is converted to text and a file containing its summary is saved in .txt file as well as *braille scripted file*.The braille file can be given as an input to refreshable braille display (rbd) machines. The audio files can also be used by a blind person for taking note of what happened during class. The screenshots are stored in a .docx file and can be used by a deaf person for revisiting concepts they might miss during classes.
 <h2 align="center"> AMAZING!!😮:star_struck:RIGHT?</h2>
 
+## Demo
+Want to know more about EduHelp. Just click the Image Below :point_down:
+<p align="center">
+  <a href="https://youtu.be/qgMU8F1QX5g"><img src="https://img.youtube.com/vi/qgMU8F1QX5g/maxresdefault.jpg" width=500 height=200></img></a>
+</p>
+
+## What Do We Think Of Implementation ?
+ - There are lot of Features which needs to be implemented for this project. To know what are those please click :point_right:[Here](https://docs.google.com/document/d/1NgP7kcJPSz0XaIq51hJkgtHk4Pm7J58N76-_0py2DTc/edit?usp=sharing).
+ - We would love to add other features in this project. It will be greatly appreciated if you contribute to this project and we ensure, it will be given a higher rating, i.e. level1, level2, level3 based on the usefulness and how inspirational your idea is, here by idea I mean just the idea and the flow of Idea in a `.md` file. If you can write your code for the idea you suggested, we will deal it in separate PR’s and with different level tags.
+
+## Tech-Stacks 💻
+- For Website: `HTML`, `CSS`
+- For core functionalities: `Python`
+- For summarizer part: `NLP`
+- For different Speech audio being clustered: `ML`
+- `Electron.js` or `Tkinter` or any other Desktop App development stack.
+- For making all the functionalities available as an API: `Flask Backend`
+
+## How to get started?
+<h3 align="center">A beginner and don't know How and Where to start???</h3>
```

Comment:
Don't capitalize the 1st letter of every word. Do it for only the 1st word. Rest all words should be in lower case.